Task- 'ActivityRecycler is absent' done

psycoAppDemon opened the pull request.

krharsh17 commented on app/src/main/java/in/tinyhouse/salesforce/Onboarding/LoginActivity.java:
```diff
@@ -1,16 +1,56 @@
+        }
+    }
+
+    public void confirmInput(View v) {
```
This method is not needed.

krharsh17 commented on app/src/main/res/layout/activity_login.xml:
```diff
@@ -5,25 +5,21 @@
     android:layout_height="match_parent"
```
Changes made in this file weren't needed. Undo all of them and then modify the java code as needed.

EditText do not need to be replaced by TextView, because user input has to be taken, and EditText is used as a standard for that.

krharsh17 commented on app/src/main/java/in/tinyhouse/salesforce/Onboarding/LoginActivity.java:
```diff
@@ -52,6 +63,7 @@ public void onClick(View view) {
                 userEmail = mEmail.getText().toString();
                 userPassword = mPassword.getText().toString();
                 loginUser(userEmail, userPassword);
+                validateE();
```
This function need not be called here. Inside the login function, call `validateE` instead of `checkEntries`

krharsh17 replied:
Rest all looks good. Also before pushing, format your code and optimize the imports. This is why the code quality check fails sometimes. Also, go through https://app.codacy.com/gh/hackslash-nitp/tinyhouse-salesforce-android/pullRequest?bid=15622094&prid=4714004 for any other quality-related changes

krharsh17 commented on app/src/main/java/in/tinyhouse/salesforce/Onboarding/LoginActivity.java:
```diff
@@ -91,6 +103,8 @@ public boolean checkEntries(String email, String password){
+        //Checking validity of the inputted email
+        validateE();
         //Checking the status of the entries by the user
         boolean status = checkEntries(email, password);
```
Instead of calling checkEntries and assigning its value to status, call validateE and assign its value to status

krharsh17 commented on app/src/main/java/in/tinyhouse/salesforce/Onboarding/LoginActivity.java:
```diff
@@ -91,6 +103,8 @@ public boolean checkEntries(String email, String password){
 
     public void loginUser(String email, String password){
+        //Checking validity of the inputted email
+        validateE();
```
Delete this call.

psycoAppDemon committed:
Update HomeActivity.java

psycoAppDemon committed:
context changed to-".home.HomeActivity" in activity_home.xml;
ui added in scanner_view.xml;
onClickListener added for "edit" in ScannerActivity.java;
dependency for RecyclerView is added

psycoAppDemon committed:
Merge branch 'master' into master
